other/run-examples.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+const { exec } = require('child_process')
+const fs = require('fs')
+const path = require('path')
+const examplesFolder = 'packages/data-point/examples/'
+
+fs.readdir(examplesFolder, (err, files) => {
+  if (err) {
+    console.log(err)
+    return
+  }
+
+  coloredLog('blue', `Executing all examples in ${examplesFolder}`, true)
+
+  files.forEach(file => {
+    const filename = examplesFolder + file
+    exec(`node ${path.resolve(filename)}`, (err, stdout, stderr) => {
+      if (err) {
+        console.error(err)
+        return
+      }
+
+      if (stdout) {
+        coloredLog('green', `stdout from ${filename}`, true)
+        coloredLog('green', stdout)
+      }
+
+      if (stderr) {
+        coloredLog('red', `stderr from ${filename}`, true)
+        coloredLog('red', stderr)
+      }
+    })
+  })
+})
+
+/**
+ * Colorize console.log()
+ * @param {enum} color - choose red, green or blue. undefined/null will be white
+ * @param {string} string - string to print
+ * @param {boolean} underline - should string be underlined
+ */
+function coloredLog (color, string, underline) {
```

Comment:
I didn't want to introduce a new dependency, but if we're OK with adding a package (maybe https://www.npmjs.com/package/chalk?) then we don't need this function.

Reply:
I'm still internally :) debating if datapoint would ever be worth porting to the frontend, adding chalk as a devDependency wouldn't be that much of a deal, but it would if it was for the production dependencies, let's leave this as is for now, it gets the job done

Reply:
btw if we did we might have to conditionally make a build where we replace request for fetch or we replace  the whole thing universal fetch module